Gruntfile.js

```diff
@@ -20,6 +20,7 @@ module.exports = function(grunt) {
     grunt.registerTask('library', ['clean:library', 'execute:ngpackagr', 'execute:typescriptTransform']);
     grunt.registerTask('styles', ['clean:styles', 'execute:less', 'usebanner:styles']);
     grunt.registerTask('assets', ['copy:fonts', 'copy:images', 'copy:css', 'copy:md']);
+    grunt.registerTask('assets:styles', ['copy:css_hpe']);
```

Comment:
I added a new task called `assets:styles` to the gruntfile as webpack was a stage too late for copying the **hpe-icons.css** across. By putting this into the `build:library` task where I have this can now be copied to the correct folder which means that the existing minify and copy to dist path processes can do the rest.

Reply:
That's fine but just rename it to something like `hpe-icons` because we have other style assets that should not be part of this task.